### ISSUE TITLE
How to exclude paths?

### ISSUE THREAD

**Issue by ericof** [codesyntax/volto-social-sharing#2]
Considering a configuration like the one bellow:
```javascript
    appExtras: [
      ...config.settings.appExtras,
      {
        match: '',
        component: SocialSharing,
      },
    ],
```
How could one exclude some paths i.e. `**/add`, `**/edit`, `**/login`?

**Comment by erral** [comment]
We are using the [appExtras feature that comes with Volto](https://github.com/plone/volto/blob/master/src/components/theme/AppExtras/AppExtras.jsx). The excludes support should be implemented there so that is available for addons like this one.

**Comment by ericof** [comment]
What about adding a check for cmsui in the component? i.e.: https://github.com/plonegovbr/volto-vlibras/blob/main/src/components/Libras.jsx#L40

**Comment by erral** [comment]
I guess you want to exclude it from non content routes right? It makes sense to me indeed.

**Comment by erral** [comment]
Can you add a PR? I'll be happy to review and release a new version

**Comment by ericof** [comment]
@erral Done! Let me know when it is released.

**Comment by erral** [comment]
I have just released version 1.2.0 with the changes.

Sorry for the version noise... we are experimenting with automatic releases through Github actions and we have not polished it yet.
